configure.ac

```diff
@@ -5191,8 +5191,6 @@ LIBS=$SAVE_LIBS
 
 # Add Rust variables, libs and utilities
 
-RUSTFLAGS=
-
```

Comment:
There is no need to remove this. That whole point of the environment variable is to override this.

Reply:
Hmm. Restoring the two lines appears to break things here, and RUSTFLAGS is once again empty if they're present. I've probably misunderstood what you were suggesting.